tcp.md

```diff
@@ -0,0 +1,10 @@
+## Setup
+
+For more details about the TCP input settings, check the [Filebeat documentation](https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-tcp).
+
+### Collecting logs from TCP
+
+To collect logs via TCP, select **Collect logs via TCP** and configure the following parameters:
+
+- Host
```

Comment:
Integrations don't really use consistent naming for these variables, so keeping it generic should be fine.

(sometimes it's `listen_address` other times it's `syslog_address`, etc)

Host and Port are probably the most important variables to set. Everything else more advanced. Since we have a link to the input docs, this should be good enough?

Reply:
We probably want to add a section about SSL settings as well, since that commonly goes with TCP.